packages/components/README.md

```diff
@@ -181,37 +181,58 @@ yarn run storybook
 
 When changes are made to the source code or .scss files for the components or the stories, the storybook instance will automatically reload.
 
-### ESLint and Prettier
-**In an effort to maintain consistent formatting, it is highly recommended to run lint-fix or lint-diff-fix on any files you've changed.**
+### Linting
```

Comment:
I have a proposed revision for the `README.md` updates. Mainly, make it a little more clear what all the commands are and remove the opinions about whats useful. Here's a screencap. What do you think?

![screencapture-github-LabKey-labkey-ui-components-new-fb-lint-config-post-post-packages-components-2020-04-21-10_04_45](https://user-images.githubusercontent.com/3926239/79893363-cf198a00-83b8-11ea-9876-ce5591fd9a46.png)

I'll DM you the patch.